tensorflow/python/framework/test_util.py

```diff
@@ -752,7 +752,8 @@ def assertArrayNear(self, farray1, farray2, err):
     """
     self.assertEqual(len(farray1), len(farray2))
     for f1, f2 in zip(farray1, farray2):
-      self.assertNear(float(f1), float(f2), err)
+      if float(f1) != float(f2):
```

Comment:
Why this change?

Reply:
@drpngx The purpose for checking `f1 == f2` is for the case where `f1, f2 = inf, inf`. `assertNear` used `math.abs(f1-f2)` which will not cover `inf, inf` case. 

I have moved the `f1 == f2` checking to `assertNear` with comments added, so that it is easy to see the intention.